### PR TITLE
[Notifications] Added new SSL notifications

### PR DESCRIPTION
**Pull request by marciocloudflare**
Also added H2 to organize by product.
Addresses [PCX-3377](https://jira.cfops.it/browse/PCX-3377)

### CODE REVIEW

File: content/fundamentals/notifications/notification-available.md [hunk 1]
```diff
@@ -292,6 +310,65 @@ Confirm your traffic is healthy: reach out to your transit providers to ensure y
...
+
+Access customers that use client certificates for mutual TLS authentication. 
+
+**Included with**
+
+Access and Cloudflare for SaaS.
+
+**What should you do if you receive one?**
+
+Upload a [renewed certificate](/ssl/ssl-for-saas/custom-certificates/).
```
Comment (kodster28) [inline content/fundamentals/notifications/notification-available.md]:
```suggestion
Upload a [renewed certificate](/cloudflare-one/identity/devices/mutual-tls-authentication/#add-mtls-authentication-to-your-access-configuration).
```

Comment (kodster28) [reply content/fundamentals/notifications/notification-available.md]: Might need to run this by Dina, but believe this should actually reference the client cert page (which is different from a custom cert) and then you would also I think need to update your access policy.

Have not done either... so just guessing based on some recent docs updates.

File: content/fundamentals/notifications/notification-available.md [hunk 2]
```diff
@@ -292,6 +310,65 @@ Confirm your traffic is healthy: reach out to your transit providers to ensure y
...
+
+Customers with dedicated certificates that want to be alerted on validation, issuance, renewal, and expiration of certificates.
+
+**Included with**
+
+When a dedicated certificate is validated, issued, renewed, or expired.
+
+**What should you do if you receive one?**
+
+Action only needed if notification is about a certificate that failed to be issued. Refer to [SSL expired or SSL mismatch errors](https://support.cloudflare.com/hc/en-us/articles/200170566#h_c1a6e78e-150d-4db6-89ab-eec7cb1ab03f) for more information.
```
Comment (kodster28) [inline content/fundamentals/notifications/notification-available.md]:
```suggestion
Action only needed if notification is about a certificate that failed to be issued. Refer to [SSL expired or SSL mismatch errors](https://support.cloudflare.com/hc/articles/200170566#h_c1a6e78e-150d-4db6-89ab-eec7cb1ab03f) for more information.
```

File: content/fundamentals/notifications/notification-available.md [hunk 3]
```diff
@@ -330,6 +407,24 @@ All Cloudflare plans.
 </div>
 </details>
 
+<details>
+<summary>Zone-level Authenticated Origin Pulls Certificate Expiration Alert</summary>
+<div>
+
+**Who is it for?**
+Customers that upload their own certificate to use with zone-level AOP to secure connections from Cloudflare to their Origin server.
```
Comment (kodster28) [inline content/fundamentals/notifications/notification-available.md]:
```suggestion
Customers that upload their own certificate to use with zone-level Authenticated Origin Pull (AOP) to secure connections from Cloudflare to their origin server.
```

File: content/fundamentals/notifications/notification-available.md [hunk 4]
```diff
@@ -330,6 +407,24 @@ All Cloudflare plans.
...
+
+**Included with**
+Authenticated Origin Pull
+
+**What should you do if you receive one?**
+Upload a renewed certificate to use for zone-level AOP.
```
Comment (kodster28) [inline content/fundamentals/notifications/notification-available.md]:
```suggestion
Upload a renewed certificate to use for [zone-level AOP](/ssl/origin-configuration/authenticated-origin-pull/set-up/). 
```

File: content/fundamentals/notifications/notification-available.md [hunk 5]
```diff
@@ -292,6 +310,65 @@ Confirm your traffic is healthy: reach out to your transit providers to ensure y
...
+
+Customers that upload their own certificate to use with hostname-level Authenticated Origin Pull (AOP) to secure connections from Cloudflare to their origin server. 
+
+**Included with**
+
+Authenticated Origin Pull.
+
+**What should you do if you receive one?**
+
+Upload a renewed certificate to use for [hostname-level AOP](/ssl/origin-configuration/authenticated-origin-pull/).
```
Comment (kodster28) [inline content/fundamentals/notifications/notification-available.md]:
```suggestion
Upload a renewed certificate to use for [hostname-level AOP](/ssl/origin-configuration/authenticated-origin-pull/set-up/#per-hostname--customer-certificates). 
```

**Comment by kodster28** [comment]
As part of this PR, could you also kill and redirect https://developers.cloudflare.com/ssl/notifications/?

**Review by kodster28 — CHANGES_REQUESTED** [review]
We need review from Dina on the expiring mTLS language before this should go live.

File: content/fundamentals/notifications/notification-available.md [hunk 6]
```diff
@@ -139,6 +145,8 @@ Success alerts require no further action. Actions for failure notifications will
 </div>
 </details>
 
+## DoS Protection
```
Comment (pedrosousa) [inline content/fundamentals/notifications/notification-available.md:148]:
```suggestion
## DDoS Protection
```

File: content/fundamentals/notifications/notification-available.md (hunk 1 above)
Comment (lgarofalo) [reply content/fundamentals/notifications/notification-available.md]: Yea, the existing reference to custom-certificates is not correct.  Your change looks correct to me, except "/ssl/" should be removed from the path for the cloudlare one documentation link, right?

https://developers.cloudflare.com/cloudflare-one/identity/devices/mutual-tls-authentication/#add-mtls-authentication-to-your-access-configuration

Comment (kodster28) [reply content/fundamentals/notifications/notification-available.md]: Howdy @lgarofalo, slightly adjusted the wording here.

I might be misunderstanding the process, but you'd want to do those two steps in separate areas of the dash, correct? First create a new cert (SSL) and then update your policy (CF one dash).

Or is the first step not required?

File: content/fundamentals/notifications/notification-available.md (hunk 6 above)
Comment (marciocloudflare) [reply content/fundamentals/notifications/notification-available.md:148]: This is actually `DoS` in the dash, @pedrosousa

Comment (pedrosousa) [reply content/fundamentals/notifications/notification-available.md:148]: I'll ask the PM if it's correct. Thanks for checking!

File: content/fundamentals/notifications/notification-available.md (hunk 1 above)
Comment (kodster28) [reply content/fundamentals/notifications/notification-available.md]: Updated based on background info from Kenny.

**Review by kodster28 — APPROVED** [review]

**Pull request merged**